cfme/tests/cli/test_appliance_console.py

```diff
@@ -585,42 +579,8 @@ def test_appliance_console_scap(temp_appliance_preconfig, soft_assert):
         initialEstimate: 1/3h
     """
 
-    command_set = ('ap', RETURN, '15', RETURN, RETURN)
-    temp_appliance_preconfig.appliance_console.run_commands(command_set, timeout=30)
-
-    with tempfile.NamedTemporaryFile('w') as f:
-        f.write(hidden['scap.rb'])
-        f.flush()
-        os.fsync(f.fileno())
-        temp_appliance_preconfig.ssh_client.put_file(
-            f.name, '/tmp/scap.rb')
-    rules = '/var/www/miq/vmdb/productization/appliance_console/config/scap_rules.yml'
-
-    temp_appliance_preconfig.ssh_client.run_command('cd /tmp/ && ruby scap.rb '
-        '--rulesfile={rules}'.format(rules=rules))
-    temp_appliance_preconfig.ssh_client.get_file(
-        '/tmp/scap-results.xccdf.xml', '/tmp/scap-results.xccdf.xml')
-    temp_appliance_preconfig.ssh_client.get_file(
-        '{rules}'.format(rules=rules), '/tmp/scap_rules.yml')    # Get the scap rules
-
-    with open('/tmp/scap_rules.yml') as f:
-        yml = yaml.safe_load(f.read())
-        rules = yml['rules']
-
-    tree = lxml.etree.parse('/tmp/scap-results.xccdf.xml')
-    root = tree.getroot()
-    for rule in rules:
-        elements = root.findall(
-            './/{{http://checklists.nist.gov/xccdf/1.1}}rule-result[@idref="{}"]'.format(rule))
-        if elements:
-            result = elements[0].findall('./{http://checklists.nist.gov/xccdf/1.1}result')
-            if result:
-                soft_assert(result[0].text == 'pass')
-                logger.info("{}: {}".format(rule, result[0].text))
-            else:
-                logger.info("{}: no result".format(rule))
-        else:
-            logger.info("{}: rule not found".format(rule))
+    temp_appliance_preconfig.appliance_console.scap_harden_appliance()
+    assert not temp_appliance_preconfig.appliance_console.scap_check_rules()
```

Comment:
Optional:

I realize this all is my mistake -- i should have done it before, but could you please make more clear that the `scap_check_rules` returns some kind of iterable of *failures*? The best would be to rename the method to something like `scap_failures`.

Thanks

Reply:
@jaryn the return value of `scap_check_rules` function itself was an iterable object, so now I done the remaining changes, please review once